<launch>
  <node pkg="grasp_suck" type="helper_services" name="helper_services_node" output="screen" clear_params="true">
    <rosparam param="home_joint">[2.834, -2.629, 1.749, -1.482, -1.457, -0.187]</rosparam>
    <rosparam param="place_joint">[3.776, -1.559, 1.602, -2.152, -1.211, -0.447]</rosparam>
  </node>
</launch>
